sceptre/cli.py

```diff
@@ -830,6 +835,21 @@ def write(var, output_format="str", no_colour=True):
     click.echo(stream)
 
 
+def _remove_response_metadata(response):
+    """
+    Removes the Response Metadata from an AWS API reponse.
+
+    :param response: The AWS response that you wish to remove the Response \
+    Metadata from.
+    :type response: dict
+    :returns: The original dict with the Response Metadata removed.
+    :rtype: dict
+    """
+    if response.get('ResponseMetadata'):
```

Comment:
`if "ResponseMetadata" in response:` is the pythonic equivalent

Reply:
You could also remove this function all together with `response.pop("ResponseMetadata", None)` as this removes the key and doesn't throw an error if the key doesn't exist.